### ISSUE TITLE
/is biomes null error

### ISSUE THREAD

**Issue by andris155** [BentoBoxWorld/Biomes#35]
####  Steps to reproduce the behavior:
1. Write /is biomes

#### Expected behavior
Command works

#### Server Information:
[Please complete the following information:]
 - Database being used (YAML, JSON, MySQL, MongoDB): []
 - OS: Ubuntu 18.04
 - Java Version: Java 11
 - BentoBox version: 1.5.2
 - Addons installed?
19.06 20:14:03 [Server] INFO Running PAPER 1.13.2.
19.06 20:14:03 [Server] INFO BentoBox version: 1.5.2
19.06 20:14:03 [Server] INFO Loaded Game Worlds:
19.06 20:14:03 [Server] INFO commands.bentobox.version.game-world
19.06 20:14:03 [Server] INFO Loaded Addons:
19.06 20:14:03 [Server] INFO AcidIsland 1.5.0
19.06 20:14:03 [Server] INFO Biomes 1.5.0.0
19.06 20:14:03 [Server] INFO Challenges 0.7.5
19.06 20:14:03 [Server] INFO Level 1.5.0
19.06 20:14:03 [Server] INFO Limits 0.2.1-SNAPSHOT
19.06 20:14:03 [Server] INFO WelcomeWarps 1.5.0
 - Other plugins?
Plugins (89): AdditionsAPI, AdminVedelem, AdvancedAchievements, antiRedstoneClock, AnvilRestrict, BentoBox, BestViewDistance, Brewery, BuntetesLogolo, ChangeSkin, ChatControl, ChatReaction*, ChestCommands, ChestShop, ChestShopNotifier*, ChestSort, Citizens, CitizensCMD, CommandSigns, CompatNoCheatPlus*, ConditionalCommands*, CoreProtect, CraftBook, CratesPlus, CrazyAuctions, CrazyAuctionsLog, CustomOreGen, Duels, EnderContainers, EntityDetection, EpicHeads, Ercgenerator, Essentials, EssentialsChat, EssentialsGeoIP, EssentialsSpawn, EventekGUI, EventRendszer, FastAsyncWorldEdit, HolographicDisplays, JukeBox, Kemkedes, Kezdes, LockettePro, Lottery, LuckPerms, mcMMO, MineMarket-Universal, MobFarmManager, Multiverse-Core, Multiverse-NetherPortals, MyCommand, NametagEdit*, NoCheatPlus, NoSpawnerChange, NoteBlockAPI, OpenInv*, PickupSpawners, PlaceholderAPI*, PlayerHeads, PlugMan, PocketGames, PomaAdditions, ProtocolLib*, ShopGUIPlus, Shopkeepers, SignEdit*, StaffChat, StaffTime, Stats, SuperLuckyBlock, Szilveszter, TiltottSzinek, TimeIsMoney*, TitleManager*, TokenManager, TokenStaff*, TreasureChestReloaded, VanillaAdditions, Vault, ViaVersion*, Votifier, VotingPlugin, WDLCompanion*, WorldEdit, WorldGuard, WorldGuardExtraFlags, WorldGuardPomaFlags, YoutubeBridge

####  Additional context
```
[15:31:55] [Server thread/INFO]: markerinh0 issued server command: /is biomes
[15:31:55] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing 'is biomes' in world.bentobox.acidisland.commands.AiCommand(is)
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:161) ~[patched_1.13.2.jar:git-Paper-632]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:734) ~[patched_1.13.2.jar:git-Paper-632]
	at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1783) ~[patched_1.13.2.jar:git-Paper-632]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1591) ~[patched_1.13.2.jar:git-Paper-632]
	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[patched_1.13.2.jar:git-Paper-632]
	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:5) ~[patched_1.13.2.jar:git-Paper-632]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:15) ~[patched_1.13.2.jar:git-Paper-632]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:108) ~[patched_1.13.2.jar:git-Paper-632]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1018) ~[patched_1.13.2.jar:git-Paper-632]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-632]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:940) ~[patched_1.13.2.jar:git-Paper-632]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:837) ~[patched_1.13.2.jar:git-Paper-632]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NullPointerException
	at world.bentobox.biomes.database.objects.BiomesObject.compareTo(BiomesObject.java:327) ~[?:?]
	at java.util.TimSort.countRunAndMakeAscending(TimSort.java:355) ~[?:?]
	at java.util.TimSort.sort(TimSort.java:220) ~[?:?]
	at java.util.Arrays.sort(Arrays.java:1515) ~[?:?]
	at java.util.stream.SortedOps$SizedRefSortingSink.end(SortedOps.java:353) ~[?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485) ~[?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[?:?]
	at world.bentobox.biomes.BiomesAddonManager.getBiomes(BiomesAddonManager.java:464) ~[?:?]
	at world.bentobox.biomes.BiomesAddonManager.getBiomes(BiomesAddonManager.java:450) ~[?:?]
	at world.bentobox.biomes.BiomesAddonManager.getBiomes(BiomesAddonManager.java:418) ~[?:?]
	at world.bentobox.biomes.panels.user.BiomesChooseGUI.<init>(BiomesChooseGUI.java:48) ~[?:?]
	at world.bentobox.biomes.commands.user.BiomesCommand.execute(BiomesCommand.java:96) ~[?:?]
	at world.bentobox.bentobox.api.commands.CompositeCommand.execute(CompositeCommand.java:246) ~[?:?]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:151) ~[patched_1.13.2.jar:git-Paper-632]
	... 14 more
```

**Comment by andris155** [comment]
[BiomesObject.zip](https://github.com/BentoBoxWorld/Biomes/files/3307426/BiomesObject.zip)

**Comment by BONNe** [comment]
So, I fix null-pointer crash in [Biomes-1.5.2.1-SNAPSHOT-b117](https://ci.codemc.org/job/BentoBoxWorld/job/Biomes/117/) and fixed issue with corrupted data in biomes database, after upgrading to 1.14 from 1.13 ([Biomes-1.5.2.1-SNAPSHOT-b118](https://ci.codemc.org/job/BentoBoxWorld/job/Biomes/118/).

Be aware, that in 1.14 world names CANNOT contain capital letters.
